guides/v2.1/coding-standards/docblock-standard-javascript.md

```diff
@@ -63,7 +65,7 @@ project.MyClass.prototype.method&nbsp;=&nbsp;function(foo)&nbsp;{
 };
 </pre>
 
-<h3 id="frontend-dev-guide-javascript-js-coding-docblock-classcomments">Class comments</h3>
+### Class comments
```

Comment:
Please append `{#frontend-dev-guide-javascript-js-coding-docblock-classcomments}` to preserve the custom id.